docs/user_guide/configuration/index.md

```diff
@@ -0,0 +1,304 @@
+
+[Celery supports various results backends](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html), including databases (such as SQLAlchemy, Django ORM), message brokers (Redis, RabbitMQ), and others. You can configure Celery to use a specific broker based on your requirements (although we suggest using Redis). However, since Merlin utilizes [Celery chords](https://docs.celeryq.dev/en/stable/userguide/canvas.html#chords) and the amqp (rpc RabbitMQ) server does not support chords, we cannot use RabbitMQ as a results backend.
+
+See the [Configuring the Broker and Results Backend](#configuring-the-broker-and-results-backend) section below for more information on configuring your results backend.
+
+## The app.yaml File
+
+In order to read in configuration options for your Celery settings, broker, and results backend, Merlin utilizes an app.yaml file.
+
+There's a built-in command with Merlin to set up a skeleton app.yaml for you:
+
+```bash
+merlin config
+```
+
+This command will create an app.yaml file in the `~/.merlin/` directory that looks like so:
+
+```yaml
+celery:
+    # see Celery configuration options
+    # https://docs.celeryproject.org/en/stable/userguide/configuration.html
+    override:
+        visibility_timeout: 86400
+
+broker:
+    # can be redis, redis+sock, or rabbitmq
+    name: rabbitmq
+    #username: # defaults to your username unless changed here
+    password: ~/.merlin/jackalope-password
+    # server URL
+    server: jackalope.llnl.gov
+
+    ### for rabbitmq, redis+sock connections ###
+    #vhost: # defaults to your username unless changed here
+
+    ### for redis+sock connections ###
+    #socketname: the socket name your redis connection can be found on.
+    #path: The path to the socket.
+
+    ### for redis connections ###
+    #port: The port number redis is listening on (default 6379)
+    #db_num: The data base number to connect to.
+
+results_backend:
+    # must be redis
+    name: redis
+    dbname: mlsi
+    username: mlsi
+    # name of file where redis password is stored.
+    password: redis.pass
+    server: jackalope.llnl.gov
+    # merlin will generate this key if it does not exist yet,
+    # and will use it to encrypt all data over the wire to
+    # your redis server.
+    encryption_key: ~/.merlin/encrypt_data_key
+    port: 6379
+    db_num: 0
+```
+
+As you can see there are three key sections to Merlin's app.yaml file: `celery`, `broker`, and `results_backend`. The rest of this page will go into more depth on each.
+
+## The Celery Section
+
+In the `celery` section of your app.yaml you can override any Celery settings that you may want to change.
+
+Merlin's default Celery configurations are as follows:
+
+??? abstract "Default Celery Configuration"
```

Comment:
Same comment as above: Is there a way to make this dynamic, eg by pointing to the actual file that will get created, so if we change the source it stays up to date?

Reply:
This specific instance with the celery settings is tricky. They're not saved to a file, instead they're stored in the app.conf object of the Celery app. I had to print the settings out and go through each of these by hand to get them in the docs here.